.codeclimate.yml

```diff
@@ -14,3 +14,4 @@ plugins:
     enabled: true
 exclude_patterns:
   - "LICENSE.md"
+  - ".github/"
```

Comment:
Why should this be excluded? These are markdown files which should be checked

Reply:
only includes bug report templates which are incorrectly flagged as having incorrect indentation while it's fine